app/src/main/java/com/pvp/app/service/NotificationReceiver.kt

```diff
@@ -18,15 +19,23 @@ class NotificationReceiver : BroadcastReceiver() {
         context: Context,
         intent: Intent
     ) {
+        val id = intent.getIntExtra("notificationId", 0)
+        val channelId = intent.getStringExtra("notificationChannelId") ?: ""
+        val title = intent.getStringExtra("notificationTitle") ?: ""
+        val text = intent.getStringExtra("notificationText") ?: ""
+
+        val channel = NotificationChannel.fromChannelId(channelId)
```

Comment:
Put channel one line above (instead of that redundant empty line above), empty line is required before condition below.